packages/api/cms-api/src/user-permissions/user-permission.resolver.ts

```diff
@@ -53,6 +53,7 @@ export class UserPermissionResolver {
     }
 
     @Query(() => [String])
+    @RequiredPermission(["userPermissions", "impersonation"], { skipScopeCheck: true })
```

Comment:
If the non-root user can't edit permissions, why is it possible load the available permissions?